packages/account-abstraction/src/BiconomyPaymasterAPI.ts

```diff
@@ -42,12 +40,25 @@ export class BiconomyPaymasterAPI implements IPaymasterAPI {
       if (result && result.data && result.statusCode === 200) {
         return result.data.paymasterAndData
       } else {
-        console.error(result.error)
-        throw new Error('Error in verifying. sending paymasterAndData 0x')
+        if (!this.paymasterConfig.strictSponsorshipMode) {
+          return '0x'
+        }
```

Comment:
Do we need to throw error here? what gets logged in SDK

Reply:
throwing below. currently signing service just throws in all negative cases so it never gets in this block. in future if they fix it we already throw result.error